src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -82,54 +81,24 @@
             .withDisplayName("gtid")
             .withType(Type.STRING)
             .withWidth(Width.LONG)
+            .withDefault("current")
             .withImportance(ConfigDef.Importance.HIGH)
             .withDescription(
                     "Single GTID from where to start reading from for a given shard."
                             + " It has to be set together with vitess.shard");
 
-    public static final Field VTCTLD_HOST = Field.create(VITESS_CONFIG_GROUP_PREFIX + "vtctld.host")
```

Comment:
Please remembere to update the configuration documentation as well

Reply:
https://github.com/debezium/debezium/blob/main/documentation/modules/ROOT/pages/connectors/vitess.adoc#vitess-adding-connector-configuration